packages/twenty-front/src/modules/object-record/record-picker/multiple-record-picker/hooks/useMultipleRecordPickerPerformSearch.ts

```diff
@@ -26,14 +28,35 @@ export const useMultipleRecordPickerPerformSearch = () => {
         forceSearchFilter = '',
         forceSearchableObjectMetadataItems = [],
         forcePickableMorphItems = [],
+        loadMore = false,
       }: {
         multipleRecordPickerInstanceId: string;
         forceSearchFilter?: string;
         forceSearchableObjectMetadataItems?: ObjectMetadataItem[];
         forcePickableMorphItems?: RecordPickerPickableMorphItem[];
+        loadMore?: boolean;
       }) => {
         const { getLoadable } = snapshot;
 
+        const paginationState = getLoadable(
+          multipleRecordPickerPaginationState.atomFamily({
+            instanceId: multipleRecordPickerInstanceId,
+          }),
+        ).getValue();
+
+        set(
+          multipleRecordPickerPaginationState.atomFamily({
+            instanceId: multipleRecordPickerInstanceId,
+          }),
+          {
+            ...paginationState,
+            currentOffset: loadMore ? paginationState.currentOffset : 0,
+            hasMore: loadMore ? paginationState.hasMore : true,
+            isLoadingMore: loadMore,
```

Comment:
do we need to maintain both isLoadingMore and isLoadingInitial? I feel isLoading is enough

why do we put it in this paginationState? can we leave the isLoading in a separate state